src/Views/FirmwareView.vala

```diff
@@ -33,7 +33,7 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
         Object (
             icon_name: "application-x-firmware",
             title: _("Firmware"),
-            description: _("Firmware updates provided by device manufacturers can improve performance and fix critical security issues.")
```

Comment:
They are not updates, and they can introduce critical safety issues and hamper performance.

Reply:
This sentence is to inform that firmware updates:
- are provided by device manufacturers (e.g. not elementary, and not a random third party)
- can improve performance
- can fix critical security issues

We can't guarantee that _all_ updates to firmware _definitely_ do those things, hence the intentional "can" language. However, communicating the potential  benefits of keeping your firmware up to date is important and the need this serves.

Reply:
It just says "firmware" twice. That doesn't help people that don't already know what firmware is.
What you then get is _of the ones that are provided by_ the flowchart goes into all sorts of directions.
There is no knowing what the opposite for example ~"by elementary OS" does, when the sentence plays out as if getting FW upgrades from "device manufacturers" is the only reason it is qualified by "can improve ___"
"Device manufacturers" isn't clear at all. Absolutely nobody that doesn't already know better will think it also includes stuff on the actual computer. 

"Can" is not the same as "may".
Is it limited to those by device manufactures?
Those kinds of questions are the ones users are faced with when it is written this way, and it is almost every string.

I didn't remove the description, but instead attempted to explain what firwmare is and how it can be upgraded.
There might be a better explanation, but let's give people a fighting chance of understanding how their systems work and what they are doing.